content/docs/0.10.x/manage/service/index.md

```diff
@@ -77,23 +73,8 @@ env:
   value: "{{ .Values.SERVICE_PLACEHOLDER_C.service.name }}"
 ```
 
-**Onboard a service:**
-
-* To onboard a service, use the [keptn onboard service](../../reference/cli/commands/keptn_onboard_service) command and provide the service name, project name (`--project` flag), and the root directory of a Helm chart (`--chart` flag): 
-
-```console
-keptn onboard service SERVICENAME --project=PROJECTNAME --chart=FILEPATH
-```
-
-* If you have already an archived Helm chart, the archive with ending `.tgz` can be referenced. In this case, the Helm chart will be stored unpacked. 
-
-```console
-keptn onboard service SERVICENAME --project=PROJECTNAME --chart=HELM_CHART.tgz
-```
-
-* Optional: In the case of using an archived Helm chart, you can achieve the same using [keptn create service](../../reference/cli/commands/keptn_create_service) and [keptn add-resource ... --all-stages](../../reference/cli/commands/keptn_add-resource), e.g.:
+* Finally, upload the Helm Chart to your service: 
```

Comment:
Just to be sure we don't break the layout 
```suggestion
1. Finally, upload the Helm Chart to your service: 
```

Reply:
I would not use the `1.` because the `*` is more generic.